+logging/getLogger.m

```diff
@@ -14,4 +14,11 @@
     obj = logging.logging(name, varargin{:});
     loggers = [loggers, obj];
   end
+  
+  deleteLogger = @() deleteLogInstance();
```

Comment:
How about the deleteLogger handler? Is there any better way of cleaning up the singleton instance?